src/rtHttpCache.h

```diff
@@ -97,7 +97,7 @@ class rtHttpCacheData
     rtError calculateRevalidationNeed(bool&,bool&);
 
     /* initiate and handle download */
-    bool handleDownloadRequest(std::vector<rtString>& headers,bool downloadBody=true);
+    virtual bool handleDownloadRequest(std::vector<rtString>& headers,bool downloadBody=true);
```

Comment:
Can we provide a test without changing the signature of handleDownloadRequest?

Reply:
will try and update